### DISCUSSION TITLE
TVA 0% uniquement pour certains clients

### DISCUSSION THREAD

**Discussion by quentinreytinas** [baimard/gestion#309] [Q&A]
Bonjour,
De temps je facture pour des clients basés en Suisse par exemple et je ne dois pas faire des factures avec TVA. Simplement si je modifie le taux de TVA dans l'onglet configuration de mon entreprise, toutes les facures sont modfifiées. Je souhaite juste de temps en temps enlever la tva. et ajouter la mention par exemple : Montant HT (exonération de TVA art.44 dir 2006/112).

Pour les remboursement de frais dois aussi facturer HT et ne pas remettre de la TVA sur de la TVA.

Merci pour votre aide !
Bien à vous,
Quentin.

**Comment by baimard** [comment]
Bonjour,
Merci d'avoir pris le temps de m'écrire et d'avoir était aussi clair dans la description.
Le fonction de taux de TVA par devis est en cours d'intégration :)

**Comment by quentinreytinas** [comment]
Merci !

-------- Message d'origine --------

**Comment by enakhala** [comment]
It will be great if in 'Company information' the name 'VAT' can be changed to 'TAX' or 'HST' or whatever applies to the users location.
